modules/storages/spec/common/storages/peripherals/nextcloud_connection_validator_spec.rb

```diff
@@ -141,11 +150,42 @@
         end
       end
     end
+
+    context "if the request returns unexpected files" do
```

Comment:
🔴 missing testcase, what happens if the `files` query returns a failure (not_found can be a good example). This opens then the question, is this maybe even another error? a la: "group folder not set up". Maybe we need an opinion from dominic here.

Reply:
I will integrate a test for the failure case for now, and we will discuss this case later.